modules/gitlab-runner/README.md

```diff
@@ -0,0 +1,176 @@
+# SETUP gitlab-runner in k8s
+
+## Provider configuration
+
+```
+terraform {
```

Comment:
please have this block included in module in new versions.tf file

Reply:
okay I add this file